### DISCUSSION TITLE
Proposed new definition of “virtual asset trading platform”: how can an "operator" also be a "virtual asset trading platform"?

### DISCUSSION THREAD

**Discussion by ghcproject** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#19] [Q&A]
Proposed new definition of “virtual asset trading platform”: how can an "operator" also be a "virtual asset trading platform"?

**Comment by FinalTwist** [comment]
I think it has to do with making the terms trading platform and operator interchangeable in the act - e.g. section 11(6).  The operator could be a separate legal entity from the trading platform, or not?

I'm curious to know if there are instances where an operator should not be considered to be a trading platform for the purposes of the Act?

**Reply by Aaron-XReg** [reply]
I don't think an operator is intended to be considered a virtual asset trading platform but rather an entity that exerts management control over a platform.

**Reply by JupiterBlock** [reply]
**Current provisions:** 

For the purposes of this Act, “_virtual asset trading platform_” means a **centralised or decentralised** digital platform —
(a) which **facilitates the exchange** of virtual assets for fiat currency or other virtual assets **on behalf of third parties** for a fee, commission, spread or other benefit; **and**
(b) which —
(i) **holds custody** of or controls virtual assets **on behalf of its clients** to facilitate an exchange; or
(ii) purchases virtual assets from a seller when transactions or bids and offers are matched in order to sell them to a buyer, and includes its owner or operator, but **does not include a platform that only provides a forum where sellers and buyers may post bids and offers and a forum where the parties trade in a separate platform or in a peer-to-peer manner.**
(3) For the purpose of the definition of “virtual asset trading platform”, where a single entity or group that controls the platform cannot be identified, the operator of the platform shall be deemed the owner of the entity under which the platform operates."

Also, see current definition of "operator":

“operator”, in relation to a virtual asset trading platform, means a person or group of persons that **exerts effective control** over the activities of a virtual asset trading platform however, in the absence of a single entity or group that exerts effective control over the platform, the operator shall be considered to be **the owner of the entity under which the platform operates**;

Under the proposed amendments, the definition of the virtual asset trading platform will not change.

It is proposed that the definition of operator changes as follows:

“owner” or “operator”, in relation to a virtual asset trading platform, means —
 (a) the single entity or group which exerts management control over the platform;
 (b) where a single entity or group which exerts management control is not identifiable, either —
  (i) the entity under which the platform operates; or
  (ii) the entity under which the platform may contract with third parties; or
 (c) where an entity or group under paragraph (a) or (b) is not identifiable, the entity or group that provides the services offered by the platform to the users of the platform."

This is intended to create more clarity via defining a step-by-step approach so that any virtual asset trading platform (which would still not include pure marketplaces or P2P) is tied to at least one entity, and is in line with the MiCA requirements in Europe. 

In addition, the proposed amendments will change the definition of "**fintech service**” (current: a service that uses innovative technology to improve, change or enhance financial services but is not a virtual asset service), to refer to "a service that uses innovative technology to improve, change or enhance **how a financial services business is conducted** but is not a virtual asset service". Financial services business is in itself defined via a reference to section 2 of the Monetary Authority Act, which means any regulated business. My guess is that the intent is to have the VASP Act clearly only concerned with the operation of regulated business and not to the technology providers of such businesses.

**Comment by CaraProvenance** [comment]
I would also interpret that operator is to allow for situations whereby the trading platform is a separate operating entity which is controlled by a further structure, i.e. the decision maker and legally binding legal entity in certain respects, as well as likely where the UBOs sit, akin to an LP/GP structure.
